projects/core/src/util/compare-equal-objects.ts

```diff
@@ -77,3 +77,104 @@ export function indexOfFirstOccurrence(
     }
   }
 }
+
+/*!
+ * Find the differences between two objects and push to a new object
+ * (c) 2019 Chris Ferdinandi & Jascha Brinkmann, MIT License, https://gomakethings.com & https://twitter.com/jaschaio
+ */
+export function diff<T, D>(objA: T, objB: D): Partial<T | D> {
```

Comment:
I cannot find usages of it anymore. @Pucek9 can we remove it?

Reply:
I've removed it